internal/operators/ocs/validations.go

```diff
@@ -115,10 +117,21 @@ func (o *ocsValidator) ValidateOCSRequirements(cluster *models.Cluster) bool {
 			}
 		}
 	}
+
+	if DiskCount%3 != 0 {
+		status = "Total disks on the cluster must be a multiple of 3"
+		o.log.Info(status)
+		err := setOperatorStatus(cluster, status)
+		if err != nil {
+			o.log.Fatal("Failed to set Operator status ", err)
```

Comment:
log.Fatal will cause the service to exit - we don't want that

Reply:
I see that we have log.Fatal elsewhere in this file and in internal/ignition/ignition.go, please fix ASAP

Reply:
fixed